ipatests/test_webui/test_automember.py

```diff
@@ -359,11 +359,11 @@ def test_rebuild_membership_users(self):
         # Remove user from group
         self.delete_record('dev1')
 
-        # Assert that user is not a member of group
+        # Assert that user was re-added to the group
+        # See https://www.port389.org/docs/389ds/design/automember-postop-modify-design.html
```

Comment:
Please add a comment stating that this behavior is expected with the plugin default setting: the entry cn=Auto Membership Plugin,cn=plugins,cn=config has a default value autoMemberProcessModifyOps: on which means that the plugin gets triggered and re-adds the entry.

Reply:
Fixed